front-end/src/components/filter/DisplayFilters.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+
+const DisplayFilters = ({ removeFilter, type, options }) => {
```

Comment:
Add PropTypes for the components props. You can check this article if you don't know how to use them.
[Link](https://reactjs.org/docs/typechecking-with-proptypes.html``)